src/WebJobs.Script.Host/Program.cs

```diff
@@ -3,6 +3,10 @@
 
 using System;
 using Microsoft.Azure.WebJobs.Script;
+using System.IO;
```

Comment:
revert this entire file (and these unecessary usings) - it doesn't look like you have any remaining changes to it.